package.json.2456964300

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
This is old package.json, right? I think this file should be completely removed.